browser-sign-in/OktaBrowserSignInUITests/Pages/ProfileScreen.swift

```diff
@@ -31,6 +36,37 @@ class ProfileScreen {
         app.tap()
         XCTAssertTrue(app.webViews.element.waitForNonExistence(timeout: 3))
     }
+    
+    func refreshToken() {
+        _ = viewTokensButton.waitForExistence(timeout: 1)
+        viewTokensButton.tap()
+        XCTAssertTrue(refreshTokenButton.exists)
+        refreshTokenButton.tap()
+        _ = app.staticTexts["Token refreshed!"].waitForExistence(timeout: 2)
+        app.buttons["OK"].tap()
+        XCTAssertNotNil(tokenLabel.value)
+    }
+    
+    func revokeToken() {
+        _ = viewTokensButton.waitForExistence(timeout: 1)
+        viewTokensButton.tap()
+        XCTAssertTrue(refreshTokenButton.exists)
+        revokeTokenButton.tap()
+    }
+    
+    func introspectToken() {
+        _ = viewTokensButton.waitForExistence(timeout: 1)
+        viewTokensButton.tap()
+        XCTAssertTrue(introspectTokenButton.exists)
+        introspectTokenButton.tap()
+        _ = app.staticTexts["Access token is active!"].waitForExistence(timeout: 1)
+        app.buttons["OK"].tap()
+    }
+    
+    func valueLabel(for identifier: String) -> String {
```

Comment:
I'd suggest you mark this function `private`, and expose the `username` label value as a property.  That would simplify the tests above.

Reply:
Good callout. I think I can get rid of this actually when i expose the `username` label value as a property